add-ons/che/che.addon

```diff
@@ -1,24 +1,25 @@
 # Name: che
 # Description: Setup and Configure Eclipse Che Template and Image Streams
 # Url: https://www.eclipse.org/che/docs/setup/openshift/index.html
+# Required-Vars: NAMESPACE
+# Var-Defaults: NAMESPACE=mini-che
+# OpenShift-Version: >=3.7.0
```

Comment:
Does that mean that the add-on cannot be installed on OpenShift 3.6?

Reply:
@l0rd Thats right. Do you think it will be an issue?

Reply:
@LalatenduMohanty @jorgemoralespou yes I think so: Che works fine at least on v3.5 and above so I would rather set `OpenShift-Version: >=3.5.0`.

And I think it is critical to change it. Otherwise, if we leave it the PR as it is it, Che addon would be unusable on default minishift (because it runs openshift v3.6).

Reply:
@l0rd Will check and fix it. The suggestion to make 3.7.0 came from https://github.com/minishift/minishift-addons/pull/68#issuecomment-355278586. I should have asked why it is needed.